ani-cli

```diff
@@ -177,26 +185,52 @@ decrypt_link() {
 }
 
 # chooses the link for the set quality
-get_video_quality() {
+get_video_link() {
 	dpage_url="$1"
-	video_links=$(decrypt_link "$dpage_url")
+	video_links=$(decrypt_link "$dpage_url")	
+	if printf '%s' "$video_links" | grep -q "mp4"; then 
+		video_url=$(get_video_quality_mp4 "$video_links")
+		idx=1
+	else
+		video_url=$video_links
```

Comment:
Is there a possibility where this would field split? Is it intentionally without quotes? If so, an explaining comment would be useful here, if not, put it in quotes.

Reply:
Oh yes!!!.. I forgor... I wonder why shellcheck overlook it...